Pcap++/header/PcapLiveDevice.h

```diff
@@ -24,6 +25,15 @@ typedef struct pcap_addr pcap_addr_t;
 */
 namespace pcpp
 {
+	/*
+	 * @class SmartPtrApiTag
+	 * Helper tag to disambiguate smart pointer API.
+	 */
+	struct SmartPtrApiTag{};
+	/**
+	 * Helper tag constant for disambuguating smart pointer API.
```

Comment:
typo: disambiguating, btw I rarely see this word, I wonder if it is precise here.

Reply:
Fixed a95c946. I think it should be better now?